### ISSUE TITLE
[Screen reader - Make Code - Import URL] User icon doesn't have alt attribute under Import Url.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2379]
**User Experience:**
Screen reader user will be affected if he/she doesn't alt attribute for image control as they won't be able to associate the context of such control.

**Test Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome
Screen reader: Narrator, JAWS, NVDA

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta#
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to import file option and select it.
6. Now, navigate to User icon appearing under Import file option.
7. Verify alt attribute of User icon.

**Actual Result:**
User icon appearing under "Import Url" doesn't have alt attribute and due to this, screen reader doesn't announce anything.

**Expected Result:**
User icon appearing under "Import Url" must have alt attribute so that screen reader user could get the context of control.

**Note:**
1. This issue also exists with Chrome + JAWS.
2. This issue also exists with Chrome + NVDA.

**MAS Reference:**
[MAS 1.1.1 - Non-text Content](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216})

[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx
) - Identify accessibility bugs before check-in and make bug fixing faster and easier.)

[MAS1 1 1_Import URL](https://user-images.githubusercontent.com/30456910/62761867-65b14f80-baa5-11e9-8fb6-610b618c1b34.png)
